Src/FluentAssertions/AlreadyFormattedString.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace FluentAssertions
+{
+    internal class AlreadyFormattedString
```

Comment:
This class is my workaround for all parameters that you used to concatenate with strings and that I now pass as argument to the string format. Because of the `StringFormatter` which wraps the string inside double quote, we were having failing tests. By introducing this new class and the associated formatter I have managed to reproduce the previous state.